### PR TITLE
Invalid validator

### PR DESCRIPTION
**Pull request by swapnakulkarni**
Need a validator that marks the field as invalid regardless of the input.
Needed for server side Validation.

### CODE REVIEW

File: lib/core/validation/validators/validator-invalid-spec.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  var invalid;
+
+  beforeEach(module('availity'));
+
+  beforeEach(inject(function (avValInvalid) {
+    invalid = avValInvalid;
+  }));
+
+  it('should never be a valid', function () {
```
Comment (ivern) [inline lib/core/validation/validators/validator-invalid-spec.js]: 'should never be valid'

**Comment by ivern** [comment]
Please point the pull request to the develop branch, not master.  Also, I think the code is fine but I also think that this validator is not intuitive at all.  I happen to know what you're trying to do, but I think a regular user of the framework will have no idea what this can be used for.

Can you put together a section on the demo/help page showing its use?

**Comment by robmcguinness** [comment]
hey @swapnakulkarni  I think a example would go along way. thanks!

**Comment by swapnakulkarni** [comment]
Yeah . I realized that does not make sense to others without example. I ll add some tomorrow



Sent with Good (www.good.com)


-----Original Message-----

**Comment by robmcguinness** [comment]
this will be handled via async validator

**Pull request closed without merging**